lib/modules/auth/widgets/header_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:podcasty/values/app_theme.dart';
+import 'package:podcasty/values/string_constants.dart';
+
+class HeaderWidget extends StatelessWidget {
+  final bool isLeftOriented;
+
+  const HeaderWidget({
+    Key? key,
+    this.isLeftOriented = true,
+  }) : super(key: key);
+
+  //Header component for Authentication module
```

Comment:
This should be a documentation comment on top of the constructor.

Reply:
Resolved